VideoWeb/VideoWeb/ClientApp/src/app/testing/mocks/MockChangesGuard.ts

```diff
@@ -1,11 +1,11 @@
 import { CanDeactivate } from '@angular/router';
 import { CanDeactiveComponent } from 'src/app/shared/guards/changes.guard';
 export class MockChangesGuard implements CanDeactivate<CanDeactiveComponent> {
-    private _flag: boolean;
+    private flag: boolean;
```

Comment:
It doesn't look like we use the ChangesGuard anywhere in this app, probably worth removing it

Reply:
This should be a separate PR